commands/Fun/compliment.js

```diff
@@ -0,0 +1,120 @@
+			"That thing you don't like about yourself is what makes you so interesting.",
+			"You're wonderful.",
+			'You have cute elbows. For reals!',
+			'Jokes are funnier when you tell them.',
+			"You're better than a triple-scoop ice cream cone. With sprinkles.",
+			'Your bellybutton is kind of adorable.',
+			'Your hair looks stunning.',
+			"You're one of a kind!",
+			"You're inspiring.",
+			"If you were a box of crayons, you'd be the giant name-brand one with the built-in sharpener.",
+			'You should be thanked more often. So thank you!!',
+			"Our community is better because you're in it.",
+			"Someone is getting through something hard right now because you've got their back.",
+			'You have the best ideas.',
+			'You always know how to find that silver lining.',
+			'Everyone gets knocked down sometimes, but you always get back up and keep going.',
+			"You're a candle in the darkness.",
+			"You're a great example to others.",
+			'Being around you is like being on a happy little vacation.',
+			'You always know just what to say.',
+			"You're always learning new things and trying to better yourself, which is awesome.",
+			'If someone based an Internet meme on you, it would have impeccable grammar.',
+			'You could survive a Zombie apocalypse.',
+			"You're more fun than bubble wrap.",
+			'When you make a mistake, you fix it.',
+			'Who raised you? They deserve a medal for a job well done.',
+			"You're great at figuring stuff out.",
+			'Your voice is magnificent.',
+			'The people you love are lucky to have you in their lives.',
+			"You're like a breath of fresh air.",
+			"You're gorgeous -- and that's the least interesting thing about you, too.",
+			"You're so thoughtful.",
+			'Your creative potential seems limitless.',
+			'Your name suits you to a T.',
+			"You're irresistible when you blush.",
+			'Actions speak louder than words, and yours tell an incredible story.',
+			'Somehow you make time stop and fly at the same time.',
+			'When you make up your mind about something, nothing stands in your way.',
+			'You seem to really know who you are.',
+			'Any team would be lucky to have you on it.',
+			"In high school I bet you were voted 'most likely to keep being awesome.'",
+			'I bet you do the crossword puzzle in ink.',
+			'Babies and small animals probably love you.',
+			"If you were a scented candle they'd call it Perfectly Imperfect (and it would smell like summer).",
+			"You're someone's reason to smile.",
+			"You're even better than a unicorn, because you're real.",
+			'How do you keep being so funny and making everyone laugh?',
+			'You have a good head on your shoulders.',
+			'Has anyone ever told you that you have great posture?',
+			'The way you treasure your loved ones is incredible.',
+			"You're really something special.",
+			"You're a gift to those around you.",
+			'You deserve better.'
+		];
+	}
+
+	async run(msg, [mentioned]) {
+		const member = mentioned || msg.member;
```

Comment:
Use default parameters for this, instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
I don't know what you mean

Reply:
He means do `async run(msg, [member = msg.member]) {`

Ngl, I'm not sure that I would have thought of that, using a previous argument to make a default like that.